javascript-linting-rules.md

```diff
@@ -0,0 +1,54 @@
+# Javascript Linting Rules
+
+The following rules should be applied to all of our javascript projects.
+
+_A project may use additional rules to ones listed here, but these would not be part of the current standard and such rules are subject to change in the future._
+
+## Current Rules
+
+These are stored in the [.eslintrc](https://raw.githubusercontent.com/holidayextras/culture/.eslintrc) in this repository.
+
+For an explanation of these rules please refer to the [ESLint documentation](http://eslint.org/docs/rules/).
+
+### TLDR
+
+For the lazy here is a brief description of the rules specified:
+
+1. `"quotes": 0`
+1. `"no-underscore-dangle": 0`  Allow variables beginning with `_`.
+1. `"curly": [2, "multi-line"]` Force curly braces except on single line statements.
+1. `"camelcase": [2, {"properties": "never"}]` Force camel case variables except on object properties.
```

Comment:
If now is the time to be discussing this, I don't think SB can adopt this rule from the off.  All of our work and current coding standards is use camelcase.  Can we strip this out for the launch?